src/stores/realTimeSessions.js

```diff
@@ -18,13 +18,13 @@ export const useSessionsStore = defineStore('sessions', {
   persist: true,
   getters: {
     currentSession (state) {
-      const currentSession = state.sessions.results.find(session => session.id === state.currentSessionId)
+      const obsPortalDataStore = useObsPortalDataStore()
+      const upcomingRealTimeSessions = obsPortalDataStore.upcomingRealTimeSessions
+      const currentSession = upcomingRealTimeSessions[state.currentSessionId]
+      // .find(session => session.id === state.currentSessionId)
```

Comment:
Need commented code?

Reply:
forgot to delete it! classic friday mood